sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.3 (Unreleased)
 
+- Enabled `TokenCachePersistenceOptions` on Node 15.
```

Comment:
Is this enough? There was never a bug on Node 15, but a bug in our tests when they got executed on Node 15 🤔